queries/c_sharp/highlights.scm

```diff
@@ -82,6 +96,15 @@
 
 (null_literal) @constant.builtin
 
+(calling_convention
+  [
+    (identifier)
+    "Cdecl"
+    "Stdcall"
+    "Thiscall"
+    "Fastcall"
+  ] @constant.builtin)
```

Comment:
`@keyword.modifier` ?

Reply:
I don't really think it's a modifier, it's an attribute/constant that determines what abi/calling convention the function uses (determines the memory layout/register usage)

Reply:
`@attribute.builtin` then? ;)